node/node_test.go

```diff
@@ -166,8 +166,10 @@ func sendPingMessage(node *Node, leader p2p.Peer) {
 		ConsensusPubKey: pubKey1,
 	}
 
-	ping1 := proto_discovery.NewPingMessage(p1)
+	ping1 := proto_discovery.NewPingMessage(p1, true)
+	ping2 := proto_discovery.NewPingMessage(p1, false)
 	_ = ping1.ConstructPingMessage()
+	_ = ping2.ConstructPingMessage()
```

Comment:
any meaningful test or check ?

Reply:
check true/false combination of different initialization